### DISCUSSION TITLE
Refused to display 'https://xxx.xxx.de/' in a frame because it set 'X-Frame-Options' to 'sameorigin'.

### DISCUSSION THREAD

**Discussion by mjonline99** [keycloak/keycloak#15354] [Q&A]
What can i do to fix this?

Refused to display 'https://xxx.xxx.de/' in a frame because it set 'X-Frame-Options' to 'sameorigin'.

Im trying to include nextcloud in an iframe.

**Comment by darius-m** [comment]
You need to change the [X-Frame-Options](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options) for the embedded service (Nextcloud?) to allow embedding in the site. It would be even better if you could also set the [frame-ancestors CSP field](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors).

But how is Keycloak related in this case?
